packages/api/src/platforms/vtex/resolvers/propertyValue.ts

```diff
@@ -2,7 +2,7 @@ import type { Resolver } from '..'
 import type { IStorePropertyValue } from '../../../__generated__/schema'
 import { getPropertyId } from '../utils/propertyValue'
 
-type Root = IStorePropertyValue
+export type Root = IStorePropertyValue
```

Comment:
the same here regarding the Input `IStorePropertyValue`. Maybe the type `StorePropertyValue` instead?